### PR TITLE
Presigned url

### PR DESCRIPTION
**Pull request by knockknockyoo**
Added test-codes
Added aws config environments

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "test": "jest"
   },
   "dependencies": {
+    "aws-sdk": "^2.1457.0",
```
Comment (n2iw) [inline package.json:11]: Is this dependency needed? Since we are doing S3 related operations in the submodule.

**Review by n2iw — CHANGES_REQUESTED** [review]
- Please take a look at the comment for the dependency
- Please update the submodule to latest 1.0.0 branch, since I've merged the PR there.

File: package.json (hunk 1 above)
Comment (knockknockyoo) [reply package.json:11]: This library requires to run the S3 operation in the submodule.

**Comment by knockknockyoo** [comment]
I've updated the submodule.

**Review by n2iw — APPROVED** [review]
LGTM

**Pull request merged**